packages/cli/src/commands/ui/create/atomic.ts

```diff
@@ -38,11 +39,9 @@ export default class Atomic extends Command {
     version: flags.string({
       char: 'v',
       description: `The version of ${Atomic.cliPackage} to use.`,
-      // default: getPackageVersion(Atomic.cliPackage), TODO: uncomment when @coveo/create-atomic added to package.json
-      default: 'latest',
+      default: getPackageVersion(Atomic.cliPackage),
     }),
   };
-  public static hidden = true;
 
   @Trackable()
```

Comment:
```suggestion
  @Trackable({
    eventName: 'ui create',
    overrideEventProperties: {framework: 'atomic'},
  })
```

This is a small change to ensure analytic data is consistent in Amplitude. For more info consult [CLI Taxonomy](https://docs.google.com/spreadsheets/d/1lexw5Nmg0qmtwaHHtBydDlJxllS_K5xSeigE4z3pGDk/edit#gid=771066228).